packages/utilities/gel-constants/README.md

```diff
@@ -0,0 +1,34 @@
+# gel-constants &middot; [![GitHub license](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](https://github.com/BBC-News/psammead/blob/latest/LICENSE) [![npm version](https://img.shields.io/npm/v/@bbc/gel-constants.svg)](https://www.npmjs.com/package/@bbc/gel-constants) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](https://reactjs.org/docs/how-to-contribute.html#your-first-pull-request)
+
+This package is a collection of variable constants which can be imported into your application.
+
+## Installation
+
+```jsx
+npm install @bbc/gel-constants --save
+```
+
+## Usage
+
+```jsx
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from 'gel-constants/breakpoints';
+
+import { GEL_GUTTER_BELOW_600PX } from 'gel-constants/spacings';
+```
+
+## Exports
+
+`/breakpoints` - GEL breakpoints, as well as typography breakpoints. These use the GEL grid sizes which can be found in their [documentation](https://www.bbc.co.uk/gel/guidelines/grid#grid-sizes).  
```

Comment:
For accessibility of links, it would be useful to have the link text to be `[GEL Grid guidelines](...)`. More descriptive both here and below.